### PR TITLE
Finalize Splash/Hero Section

### PR DESCRIPTION
**Pull request by ptwu**
### Summary
This PR implements a text-editor effect to the subtitle of "Full Stack Developer and CS Student" and introduces a Projects section that will be added on to later with my past dev projects. It also implements a "Learn More" button which auto-scrolls to this section using JS.

### Screenshots
**Blinking Text/Typing Effect on Subtitle**
GIF of blinking text effect

**Learn More Button Leading to Projects Section**
Learn More Button

### Test Plan 
Run `mvn package appengine:run` into the Cloud Shell to view a test server of the site by clicking Web Preview on the top right. You can play around with the Learn More button and notice that the blinking text effect is present.

### Breaking Changes
N/A

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -5,35 +5,57 @@
...
-    <link rel="icon" href="favicon.ico" type="image/x-icon">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="favicon.ico" type="image/x-icon" />
     <script async src="script.js"></script>
   </head>
   <body>
     <div class="NavbarWrap">
-      <a href="#" id="logoButton">
-        <img src="/assets/images/logo.png" alt="Peter Wu logo" width="72px" />
+      <a href="#" id="LogoButton">
```
Comment (chaowangzj) [inline portfolio/src/main/webapp/index.html:14]: We usually don't capitalize the first letter in id or class.

Comment (chaowangzj) [inline portfolio/src/main/webapp/index.html:14]: It is good to keep the naming consistent, so I just put it as FYI: Google CSS style guide recommends to use all lowercases in html and separate words with dashes.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -5,35 +5,57 @@
...
+        <h2 class="Subtitle">
+          &lt;Full Stack Developer /&gt; and CS Student
+          <div id="BlinkingText">_</div>
+        </h2>
+        <p class="DescText">
+          I'm a freshman at Cornell who likes to build things that make marked
+          improvements on our lives.
         </p>
-        <p>Click here to get a random fact about Peter:</p>
-        <button onclick="addRandomFact()">Click Me</button>
```
Comment (chaowangzj) [inline portfolio/src/main/webapp/index.html:27]: I see you keep the function in the js. Do you intent to use it somewhere?

Comment (ptwu) [reply portfolio/src/main/webapp/index.html:27]: Yes, I plan on modifying this with a typewriter effect that displays each fact in the hero section. I will do this after I finish the projects section.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (ptwu) [reply portfolio/src/main/webapp/index.html]: Done - just refactored to fit the style guidelines. It seems like interns don't have access to the internal version of the style guide, so I've been referring to the one on google.github.io.

**Pull request merged**